pkg/splunk/enterprise/configuration.go

```diff
@@ -484,7 +491,7 @@ func updateSplunkPodTemplateWithConfig(podTemplateSpec *corev1.PodTemplateSpec,
 	configMapVolDefaultMode := int32(corev1.ConfigMapVolumeSourceDefaultMode)
 
 	// add inline defaults to all splunk containers
-	if spec.Defaults != "" {
+	if spec.Defaults != "" && instanceType != SplunkMonitoringConsole {
```

Comment:
Why aren't defaults being added to MC? Can the reason be provided as part of the comment above?

Reply:
For MC we don't need to supply it with the spec defaults of the custom resource